docs/articles/documentation/getting-started/index.md

```diff
@@ -64,6 +66,10 @@ testcafe safari test1.js
 
 TestCafe will automatically open the chosen browser and start the test execution within it.
 
+> Important! Make sure to keep the browser tab that is running tests active. Do not minimize the browser window.
```

Comment:
Is it only for the current example? If i have 2 browsers i can't keep their active simultaneously.